webpack/assets/javascripts/react_app/API.js

```diff
@@ -0,0 +1,11 @@
+import ServerActions from './actions/ServerActions';
+
+export default {
+    getStatisticsData(url) {
+        $.getJSON(url)
+            .success(
+                rawStatistics => ServerActions.receivedStatistics(rawStatistics)
+            )
+            .error(error => console.log(error));
```

Comment:
I would also assume error handling could be extracted to a generic function? so we dont have to repeat for every api call?